src/index.html

```diff
@@ -13,6 +25,46 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article>
+      <div
+        data-qa="card"
+        class="product-card"
+      >
+        <img
+          class="product-card__image"
+          src="/src/images/imac.jpeg"
+          alt="Product image"
+        />
+        <h2 class="product-card__name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+          <p class="product-card__code">Product code: 195434</p>
+        </h2>
+
+        <div class="product-card__stars">
+          <div class="product-card__stars-star"></div>
+          <div class="product-card__stars-star"></div>
+          <div class="product-card__stars-star"></div>
```

Comment:
This part of the task is incomplete, you need to add the modifier class and use styles from the previous task 
<img width="528" alt="image" src="https://github.com/user-attachments/assets/c49a2763-8686-4fe3-9cec-5f4c15ce3a80">